crates/test-utils/src/bin/dump-katana.rs

```diff
@@ -35,11 +37,19 @@ async fn main() {
             .dump_state()
             .expect("Failed to call dump_state on Katana state");
 
-        let state = serde_json::to_string(&dump_state).expect("Failed to serialize state");
-
         // Dump the state
         std::fs::create_dir_all(".katana/").expect("Failed to create Kakata dump dir");
-        std::fs::write(".katana/dump.json", state).expect("Failed to write dump to .katana/dump.json");
+
+        let katana_dump_path = String::from(".katana/dump.json");
+        let katana_dump_file = File::options()
+            .create_new(true)
+            .read(true)
+            .write(true)
+            .append(false)
+            .open(katana_dump_path)
+            .expect(format!("Failed to open file {}", katana_dump_path));
```

Comment:
Should be `&str` I think

Reply:
fixed [here](https://github.com/kkrt-labs/kakarot-rpc/pull/562/commits/60668d27a0a0628c0a0929e1024430cd9c1cc233).